### PR TITLE
Validator media

### PR DESCRIPTION
**Pull request by ptrus**
Two fixes:
First: rename ValidatorMedia fields to match the names of the one in the Metadata Registry
- website_link -> url, email_address -> email, twitter_acc -> twitter... 
- before only `name` was matching, therefore the API response always only included name, because other fields couldn't be automatically json decoded
- we could also keep the fields named as they are, and manually convert from metadata registry field names, to ValidatorMedia field names, but probably no reason for that
 
Second: Add `logo_url` field to the `chain.entities` table (and `ValidatorMedia`) and update the `MetadataRegistry` analyzer to fetch the `logo_url` from the keybase handle (this is the same way as Oasisscan uses to fetch the logo):
- it makes sense doing this on the backend to avoid frontend having to issue lookup requests to keybase to obtain the log_urls from the keybase handle

See updated db state with the fix:

```
                      id                      |                    address                     |                                                                                                      meta                                                                                                      |                                             logo_url
----------------------------------------------+------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------
 kfr2A6K6TlvhQm4nz88Hczzkd2Aq5PlkxSpnmUUBAFs= | oasis1qr0jwz65c29l044a204e3cllvumdg8cmsgt2k3ql | {"v": 1, "url": "https://staking.fund", "name": "Staking Fund", "email": "go@staking.fund", "serial": 1, "keybase": "StakingFund", "twitter": "StakingFund"}                                                   | https://s3.amazonaws.com/keybase_processed_uploads/d48739023a250815c4ac564c9870ec05_360_360.jpg
 KzotvVsgmTBNDsyHdv0CZzQBBi3r9lcKxWuCD0uFQcY= | oasis1qpgk5ahlvr97rj76l9e75fwedwwhg25zxqep8jj4 | {"v": 1, "url": "https://terminet.io/", "name": "Terminet", "email": "support@terminet.io", "serial": 2, "keybase": "EBB3287F213D3847"}                                                                        |
 QgG1hFl4On84uVIrd++d3HxwoirdopH6oRmpltAIxxQ= | oasis1qrfeessnrnyaggvyvne52aple2f8vaw93vt608sz | {"v": 1, "url": "https://keybase.io/julia_ju", "name": "Julia-Ju", "serial": 1, "keybase": "julia_ju"}                                                                                                         | https://s3.amazonaws.com/keybase_processed_uploads/127dc934642b7ae27de0a2314cba6605_360_360.jpg
 aiTgGyYB2l4uAMG93Ajq5S4EXPIRkYDg1ICLjWD45Ck= | oasis1qrugz89g5esmhs0ezer0plsfvmcgctge35n32vmr | {"v": 1, "url": "https://www.validatrium.com", "name": "Validatrium.com", "email": "validatrium@gmail.com", "serial": 2, "keybase": "validatrium", "twitter": "validatrium"}                                   | https://s3.amazonaws.com/keybase_processed_uploads/ff96631d063abf8c89ebeda32d884d05_360_360.jpg
 vCyhq/W+qN9tgLZSIn1MWFdJezZaa0Jm5VMtAOSRAi8= | oasis1qp690sk0vrxhreg9u08zazxt5p62qyvr6q8pg5zc | {"v": 1, "url": "https://senseinode.com", "name": "Sensei node", "email": "federico@senseinode.com", "serial": 3, "keybase": "fefehof", "twitter": "senseinode"}                                               | https://s3.amazonaws.com/keybase_processed_uploads/7a6a7bb56e638c0a58fa1c1b1cb5d005_360_360.jpg
 7UFm7kLE8iAxEY7ABKWRYSgwkZ18g/U3e9xxdqBCUgM= | oasis1qpn9vaeh742wypj83v83h2e4elr8p4jywullc5f8 | {"v": 1, "name": "ptrus", "email": "peter@oasisprotocol.org", "serial": 1}                                                                                                                                     |
 /Foi5kZdBc70HPmlguvY6AzBILNrNiPDgKQLOITuIEI= | oasis1qps9drw07z0gmh5z2pn7zwl3z53ate2yvqf3uzq5 | {"v": 1, "name": "cherkes", "serial": 1, "keybase": "cherkes", "twitter": "CryptoMad77"}                                                                                                                       | https://s3.amazonaws.com/keybase_processed_uploads/b8b4a8365e10e870a60455ed1da57f05_360_360.jpg
 BBba0XfI4WSTroE+/YGcFthxbEC5jQJhWhl0jIBwke0= | oasis1qq2kqzr4q942x44st97n66nmmyh7dhsuvsqyc22u | {"v": 1, "url": "https://my.entity/url", "name": "Aptemuyc🌹", "email": "lazarev.zahar@yandex.ru", "serial": 1, "keybase": "aptemuyc", "twitter": "g5sliqjq2g92jet"}                                            |
 trheWccoAX4yy4FVnTBdt4u83GGHbgWs1gy6OBifClc= | oasis1qzl58e7v7pk50h66s2tv3u9rzf87twp7pcv7hul6 | {"v": 1, "url": "https://blockscape.network/", "name": "blockscape", "email": "blockscape-dev@mwaysolutions.com", "serial": 1, "keybase": "blockscapeval", "twitter": "BlockscapeLab"}                         | https://s3.amazonaws.com/keybase_processed_uploads/0df6d91e7e57b42a7e17b2cc62541905_360_360.jpg
 BkBqXMdd10xDKn7b/iIXycHLBaroci6xoFxoEN1kQ+A= | oasis1qqqn56k79st0zy4060m0wvmec76vd3pl6czmg90a | {"v": 1, "url": "https://blockchain.ubc.ca/", "name": "Oasis@UBC", "email": "zwang@ece.ubc.ca", "serial": 1, "twitter": "blockchainubc"}                                                                       |
```

### CODE REVIEW

File: analyzer/metadata_registry/metadata_registry.go [hunk 1]
```diff
@@ -2,21 +2,27 @@ package metadata_registry
...
 	"context"
+	"encoding/json"
 	"fmt"
+	"io"
+	"net/url"
 	"time"
 
 	registry "github.com/oasisprotocol/metadata-registry-tools"
 
-	staking "github.com/oasisprotocol/nexus/coreapi/v22.2.11/staking/api"
```
Comment (pro-wh) [inline analyzer/metadata_registry/metadata_registry.go:10]: we've been keeping these separate \\:

**Review by pro-wh — APPROVED** [review]
typos:

fileds -> fields
oasiscan -> oasisscan

**Comment by pro-wh** [comment]
interesting, these validator field names date back to #67

File: analyzer/metadata_registry/metadata_registry.go (hunk 1 above)
Comment (ptrus) [reply analyzer/metadata_registry/metadata_registry.go:10]: Fixed

**Pull request merged**